packages/solana/src/types/nft.ts

```diff
@@ -19,7 +19,7 @@ export const CommonTokenInput = z
  * @internal
  */
 export const CommonTokenOutput = CommonTokenInput.extend({
-  id: z.string(),
+  id: z.string().optional(),
```

Comment:
Oh right... There's no address for lazy minted tokens

Reply:
Unfortunately